src/pudl/metadata/resources/eia.py

```diff
@@ -1045,6 +1045,35 @@
         "sources": ["eia860", "eia923"],
         "etl_group": "outputs",
     },
+    "core_eia__yearly_so2_control_equipment": {
```

Comment:
I think it may be too soon to add this here if we haven't added it to the harvesting process yet.

Reply:
I'm using it to encode the table in preparation for harvesting, rather than adding the `_core` table here as the `_core` table won't be persisted to the database. It's jumping the gun a bit but I don't want to finish cleaning here without standardizing dtypes. Thoughts?

Reply:
Switched this up a bit to reflect our likely pause on harvesting work in the near future.